Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,62 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+joint operational direction of the Node.js Technical Steering Committee (TSC)
+and Node.js Foundation Community Committee (CommComm).
+
+## Organization Roles
+
+### Owners
+
+The Node.js TSC Director, TSC Facilitator, and the two elected individual
+member representatives shall be granted Owner permissions within the Node.js
+GitHub Organization.
+
+Additional TSC, CommComm, and Core Technical Committee (CTC) members may be
+granted Owner permissions by request. Assuming there are no objections from any
+current voting TSC or CommComm member, such requests are approved automatically
+after 72 hours. If any objection is made, the request may be moved to a single
+vote in which a simple majority of both the TSC and CommComm is required to
+approve. Such requests must be posted as issues to either the nodejs/TSC or
+nodejs/community-committee repositories.
+
+Upon the completion of the TSC Director, TSC Facilitator, or elected individual
+member representative's terms, their Owner permissions within the GitHub
+organization shall be maintained *unless* the individual is also no longer a
+voting member of either the TSC or Community Committee.
```

Comment:
this and the below paragraph seem to contradict each other if an elect individual member is also on CTC